sirepo/sim_data/raydata.py

```diff
@@ -30,5 +30,5 @@ def _input_files():
                 if v:
                     yield cls.lib_file_name_with_model_field('inputFiles', k, v)
         return [
-            data.models.analysisAnimation.notebook,
+            data.models.scans.catalogName + '.zip',
```

Comment:
Make a function in SimData that is
```python
@classmethod
def raydata_notebook_zip_filename(cls, data):
    return data.models.scans.catalogName + '.zip'
```

Reply:
60964def15947627c953fcd18a3c6393619dfd97